kusto/doc.go

```diff
@@ -56,6 +56,36 @@ and accepts only string constants for arguments.
 		panic("add error handling")
 	}
 
+# Querying with parameters
+
+Users will sometimes want to use query parameters (Read more here - https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/queryparametersstatement?pivots=azuredataexplorer).
+In this case, the StatementBuilder and QueryParameters structs should be used.
+
+	dt, _ := time.Parse(time.RFC3339Nano, "2020-03-04T14:05:01.3109965Z")
+	// Query our database table "systemNodes" for the CollectionTimes and the NodeIds.
+	iter, err := client.Query(ctx, "database", kql.NewStatementBuilder("systemNodes | where CollectionTime == time and NodeId == id"),
+		[]kusto.QueryOption{kusto.QueryParameters(*kql.NewStatementQueryParameters().
```

Comment:
That's the wrong way to call the method, you don't need to create the array. Try it out and see